frontend/css/wi-volunteer-management-public.css

```diff
@@ -80,4 +80,33 @@
 
 #wivm-sign-up-form .field-error {
 	border: 2px solid #990000;
+}
+
+/* Widget Styles */
+.widget-volunteer-opp-info {
```

Comment:
I can see where scrolling could frustrate users. Especially when you're scrolling inside of a page that also scrolls. Great feedback! Do we want to get rid of the max-height in addition to the scrolling?